packages/backend/src/helpers/authentication.ts

```diff
@@ -72,6 +72,10 @@ const authentication = shield(
       '*': or(isAuthenticated, isAdminQuery),
       getTable: or(isAuthenticated, isAdminQuery, isViewKey),
       getAllRows: or(isAuthenticated, isAdminQuery, isViewKey),
+      getFlowOwner: isAdminQuery,
+      getExecutionOwner: isAdminQuery,
+      getTableOwner: isAdminQuery,
+      searchUsersByEmail: isAdminQuery,
```

Comment:
food for thought: in case we forget to add future admin endpoints here, we can have add sth like
```
type AdminQuery {
  getFlowOwner(flowId: String!): User!
  getExecutionOwner(executionId: String!): User!
  getTableOwner(tableId: String!): User!
  searchUsersByEmail(query: String!): [User!]!
}
```
in schema.graphql and set the permissions for AdminQuery here to be '*': isAdminQuery

Reply:
Jury-rigged this a little bit. LMK what you think